CLAUDE.md

```
@@ -174,8 +174,10 @@ Turborepo monorepo: `apps/frontend` (Next.js 15/React 19), `apps/backend` (NestJ
 
 ## Deployment
 
-**Frontend**: Vercel (https://tenantflow.app) - auto-deploy from main
-**Backend**: Railway (https://api.tenantflow.app) - Dockerfile
+## Deployment
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)